docs/architecture.md

```diff
@@ -28,7 +28,7 @@ The Feather game state is defined in the `Game` struct, which lives in `crates/c
 This struct contains the `World` (blocks) and the `Ecs` (entities). It also provides
 methods for common tasks, like "spawn entity" or "remove entity" or "get block."
 
-Note that entities in the ECS correspond either to Minecraft entities, like players or zombies,
+Note that entities in the ECS correspond either to Minecraft entities, like players or zombies
```

Comment:
Either leave the comma or put the whole statement in parenthesis. The original was correct.